tasks/src/main/java/ru/sberbank/school/task07/FileParser.java

```diff
@@ -0,0 +1,22 @@
+package ru.sberbank.school.task07;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+/**
+ * Интерфейс парсера файла.
+ * <p>

```

Comment:
&lt;p&gt; tag should be placed immediately before the first word, with no space after.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1609591082>*